src/omv/modules/py_tf.c

```diff
@@ -655,7 +691,7 @@ STATIC mp_obj_t int_py_tf_segment(bool detecting_mode, uint n_args, const mp_obj
     py_tf_input_data_callback_data.img = image;
     py_tf_input_data_callback_data.roi = &roi;
 
-    py_tf_segment_output_data_callback_data_t py_tf_segment_output_data_callback_data;
+    mp_obj_t py_tf_segment_output_data_callback_data;
```

Comment:
```suggestion
    mp_obj_t py_tf_detect_output_data_callback_data;
```

I know it's just a variable name, but it seems you want to match the function name.

Reply:
Done

Reply:
Wait, for the following suggestion, I just renamed everything to remove the data piece. This should match the callback name. So, it shouldn't be detect as the callback name is segment.